### PR TITLE
Robbie Tank and Default Magazine change, Default Fuel Adjustment

### PR DESCRIPTION
**Pull request by BradMick**
Add Robbie Tank
Changes default magazine from 1200rd to 300rd
Adds garage toggle option for Magazine
Adjusts default fuel to provide a 5% power margin for the aircraft

### CODE REVIEW

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fuel/fn_fuelSet.sqf [hunk 1]
```diff
@@ -22,13 +22,34 @@ Author:
...
+} else {
+	_maxTotFuelMass = _maxFwdFuelMass + _maxAftFuelMass;
+	_heli setVariable ["fza_sfmplus_maxTotFuelMass", _maxTotFuelMass];
+
+	_totFuelMass = _maxTotFuelMass * _percentFuel;
+	_fwdFuelMass = [_totFuelMass / 2, 0, _maxFwdFuelMass] call BIS_fnc_clamp;
+	_aftFuelMass = _totFuelMass - _fwdFuelMass;
+};
+
+systemChat format ["Tot Fuel Mass %4 / Fwd Fuel Mass %1 / Ctr Fuel Mass %2 / Aft Fuel Mass %3", _fwdFuelMass, _ctrFuelMass, _aftFuelMass, _totFuelMass];
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fuel/fn_fuelSet.sqf:53]: Can you remove this please?

**Review by mattysmith22 — CHANGES_REQUESTED** [review]
Looks great! There's just a couple small issues that need to be fixed

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/engine/fn_engineHandleControl.sqf [hunk 2]
```diff
@@ -55,7 +54,6 @@ switch(_control) do {
 			[_heli] spawn fza_fnc_fxLoops;
 			["fza_ah64_battery", 0.1] spawn fza_fnc_playAudio;
 		} else {
-			_heli animateSource["plt_batt", 0];
```
[the remark this answers is not in the document]
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/engine/fn_engineHandleControl.sqf:58]: No idea

**Review by mattysmith22 — APPROVED** [review]
Good job!

**Pull request merged**